README.md

```diff
@@ -42,6 +42,11 @@ Following additional parameters can be used:
 
         def hello():
             print("hello world")
+
+    .. jupyter-execute::
+        :hide-code:
+
+        print("This is for jupyter-sphinx")
```

Comment:
don't need this example here -- these are just meant to explain the different file types (you can rebase out this commit -- thanks!)